From 6329af20141a42f0aacdf65e56f8d4d11a27715b Mon Sep 17 00:00:00 2001
From: Illia Vysochyn <bitfun6@gmail.com>
Date: Sat, 22 Oct 2022 23:30:29 +0200
Subject: [PATCH] Remove symbols check

---
 configure.ac | 38 --------------------------------------
 1 file changed, 38 deletions(-)

diff --git a/configure.ac b/configure.ac
index 8a8066a..acbb6b0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -88,44 +88,6 @@ if test "x$enable_fonts" != xfalse; then
   else
       AC_DEFINE_UNQUOTED(HAVE_CONSOLECHARS, $CONSOLECHARS)
   fi
-
-  if test x$CONSOLECHARS != x -o x$CONSOLECHARS != x'"$CONSOLECHARS"'; then
-       if test x$SETFONT != x -o x$SETFONT != x'"$SETFONT"'; then
-
-	  dnl Now look for the console fonts directory
-	  AC_CHECK_FILES([/usr/lib/kbd/consolefonts /usr/share/consolefonts])
-
-	  if test "x$ac_cv_file__usr_lib_kbd_consolefonts" = "xno"; then
-	      if test "x$ac_cv_file__usr_share_consolefonts" = "xno"; then
-		  AC_MSG_WARN([ 
-
-*** You do not appear to have a consolefonts directory in a standard location
-*** (/usr/lib/kbd or /usr/share), even though you appear to have the
-*** consolechars and/or setfont command.  The matrix font for the console
-*** will not be installed.  This means you will not be able to use the
-*** matrix console font (and the -l command line switch) unless the font
-*** is located in your current directory when you run CMatrix.
-])
-	      fi
-	  fi
-       fi
-  fi
-
-  AC_PATH_PROG(MKFONTDIR, mkfontdir, "", $PATH:/usr/bin:/usr/bin/X11:/usr/local/bin/X11:/usr/X11R6/bin:/usr/local/bin:/sbin:/usr/sbin)
-  AC_CHECK_FILES([/usr/lib/X11/fonts/misc /usr/X11R6/lib/X11/fonts/misc])
-
-  if test "x$ac_cv_file__usr_lib_X11_fonts_misc" = "xno"; then
-      if test "x$ac_cv_file__usr_X11R6_lib_X11_fonts_misc" = "xno"; then
-	  AC_MSG_WARN([ 
-
-*** You do not appear to have an X window fonts directory in the standard
-*** locations (/usr/lib/X11/fonts/misc or /usr/X11R6/lib/X11/fonts/misc). The
-*** mtx.pcf font will not be installed.  This means you will probably not
-*** be able to use the mtx fonts in your x terminals, and hence be unable
-*** to use the -x command line switch.  Sorry about that...
-])
-	  fi
-      fi
 fi
 
 dnl Parse any configure options
-- 
2.38.0

